app/models/healthcheck.rb

```diff
@@ -0,0 +1,42 @@
+class Healthcheck
+  def name
```

Comment:
This should probably return a symbol like the others, but also I'm not convinced we even need this method.